ReactAndroid/src/main/java/com/facebook/react/views/drawer/ReactDrawerLayoutManager.java

```diff
@@ -90,6 +93,21 @@ public void setDrawerLockMode(ReactDrawerLayout view, @Nullable String drawerLoc
   }
 
   @Override
+  public void setElevation(ReactDrawerLayout view, float elevation) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      // Facebook is using an older version of the support lib internally that doesn't support
+      // setDrawerElevation so we invoke it using reflection.
+      // TODO: Call the method directly when this is no longer needed.
+      try {
+        Method method = ReactDrawerLayout.class.getMethod("setDrawerElevation", float.class);
+        method.invoke(view, PixelUtil.toPixelFromDIP(elevation));
+      } catch (Exception e) {
+        // Ignore errors when the method is not present in older versions of support lib.
```

Comment:
Can you add a warning for the error case (`FLog.e`). Somebody might try to use this method internally and be suprised about it not working.